controllers/persistentvolumeclaim_controller.go

```diff
@@ -127,3 +61,18 @@ func (r *PersistentVolumeClaimReconciler) SetupWithManager(mgr ctrl.Manager) err
 		For(&corev1.PersistentVolumeClaim{}).
 		Complete(r)
 }
+
+func removeTopoLVMFinalizer(pvc *corev1.PersistentVolumeClaim) ([]string, bool) {
+	removed := false
+	// Due to bug #310, we need to de-dup TopoLVM finalizers.
```

Comment:
This comment does not describe what the code does. please update or remove it.

Reply:
I revised the comment.